Maple2.Server.Game/Manager/Field/FieldManager.State.cs

```diff
@@ -171,6 +172,10 @@ public partial class FieldManager {
         return fieldItem;
     }
 
+    public int RemoveMeNextLocalId() {
+        return NextLocalId();
+    }
```

Comment:
If NextLocalId is private, there must be reason why, but you if you need for removeme, then just make it public? 🤔

Reply:
Woops, that was for the item spawn debug printing. I put removeme in the name so either I would remove it myself or I'd get barked at in review to remove it if I forgot lol